accounts/src/account_service.rs

```diff
@@ -0,0 +1,302 @@
+                        )
+                        .await;
+                    match accounts {
+                        Ok(response) => {
+                            fetch_accounts = response.value;
+                            updated_slot = response.context.slot;
+                            break;
+                        }
+                        Err(e) => {
+                            // retry
+                            log::error!("Error fetching all the accounts {e:?}, retrying");
+                            continue;
+                        }
+                    }
+                }
+                for (index, account) in fetch_accounts.iter().enumerate() {
+                    if let Some(account) = account {
+                        self.account_store
+                            .initilize_account(AccountData {
+                                pubkey: accounts[index],
+                                account: account.clone(),
+                                updated_slot,
+                            })
+                            .await;
+                    }
+                }
+            }
+        }
+        log::info!("{} accounts successfully fetched", accounts.len());
+        Ok(())
+    }
+
+    pub fn process_account_stream(
+        &self,
+        mut account_stream: AccountStream,
+        mut block_stream: BlockStream,
+    ) -> Vec<AnyhowJoinHandle> {
+        let this = self.clone();
+        let processed_task = tokio::spawn(async move {
+            loop {
+                match account_stream.recv().await {
+                    Ok(account_notification) => {
+                        this.account_store
+                            .update_account(
+                                account_notification.data,
+                                account_notification.commitment,
+                            )
+                            .await;
+                    }
+                    Err(tokio::sync::broadcast::error::RecvError::Lagged(e)) => {
+                        log::error!("Account Stream Lagged by {}", e);
+                        continue;
+                    }
+                    Err(tokio::sync::broadcast::error::RecvError::Closed) => {
+                        log::error!("Account Stream Broken");
+                        break;
+                    }
+                }
+            }
+            bail!("Account Stream Broken");
+        });
+
+        let this = self.clone();
+        let block_processing_task = tokio::spawn(async move {
+            loop {
+                match block_stream.recv().await {
+                    Ok(block_notification) => {
+                        if block_notification.commitment_config.is_processed() {
+                            // processed commitment is not processed in this loop
+                            continue;
+                        }
+                        let commitment = Commitment::from(block_notification.commitment_config);
```

Comment:
nit: could be
`let commitment = block_notification.commitment_config.into();`

Reply:
Into should be avoid according to solana standards